values.schema.json

```diff
@@ -80,7 +80,11 @@
                             ]
                         },
                         "tolerations": {
-                            "type": ["null", "string"]
+                            "type": [
+                                "null",
```

Comment:
I think that it is still needed to avoid breaking preexisting configuration e.g in the case where someone has set it explicitly to `null`.  I believe validation is done for the following commands `install`, `upgrade`, `lint`, and `template`.